build.gradle

```diff
@@ -18,7 +18,9 @@ buildscript {
         maven {
             url 'https://maven.fabric.io/public'
         }
-
+        maven {
+            url 'https://developer.huawei.com/repo/' // AREngine from Huawei
```

Comment:
Would it be easily possible to add this just to the AREngine build variant(s) instead of "allprojects" & "buildscript"?

Reply:
Done